sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/DirectoryAsyncClient.java

```diff
@@ -302,13 +302,13 @@ public DirectoryAsyncClient getSubDirectoryClient(String subDirectoryName) {
      * @param recursive Specifies operation should apply to the directory specified in the URI, its files, its subdirectories and their files.
      * @return {@link HandleItem handles} in the directory that satisfy the requirements
      */
-    public Flux<HandleItem> getHandles(Integer maxResult, boolean recursive) {
+    public Flux<HandleItem> listHandles(Integer maxResult, boolean recursive) {
```

Comment:
This may be tracked in a separate issue but looks like there are few things different for this client from the rest:
1. Flux instead of PagedFlux
2. Missing annotations like `@ServiceClient` and `@ServiceMethod`
3. `WithResponse` variations of these public APIs

Reply:
I think there's an open design issue for the return type on non-paged listing operations (listBlocks on blobs would be another examples). In any case, I don't think Files has been updated to PagedFlux and with response for any apis yet.

Reply:
1. The PageFlux and PageIterable changes will be handled in another PR.
For the list handles specifically, will consist with Blob for now, which return Mono<T> and T respectively for async and sync.
2. The annotation changes will address in Shawg's checkstyle PR.
3. Tracing PR has already merged from most recent commits. It is there already.